docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -1919,6 +1920,31 @@ SELECT regexp_replace('aBc', '(b|d)', 'Ab\\1a', 'i');
 
 Additional examples can be found [here](https://github.com/apache/datafusion/blob/main/datafusion-examples/examples/regexp.rs)
 
+### `regexp_extract`
+
+Extract a specific group matched by [regular expression](https://docs.rs/regex/latest/regex/#syntax). If the regex did not match, or the specified group did not match, an empty string is returned..
+
+```
+regexp_extract(str, regexp[, idx])
```

Comment:
this seems like the index is optional while it is required

Reply:
fixed